externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -177,3 +181,18 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
 
   override protected def isServer: Boolean = false
 }
+
+object SparkSQLSessionManager {
+  private val sessionManager = new AtomicReference[SparkSQLSessionManager]
+
+  private def setSessionManager(manager: SparkSQLSessionManager): Unit = {
+    sessionManager.set(manager)
+  }
+
+  def getSparkSession(uuid: String): SparkSession = {
+    sessionManager.get()
```

Comment:
how about get from object `SparkSQLEngine`.

Reply:
we need a static method here, and call the method and return sql spark session on the python side. how can we get the sql spark session from `SparkSQLEngine`?

Reply:
```
  SparkSQLEngine.currentEngine.map(_.backendService.sessionManager)

```

Reply:
with that, SparkSQLSessionManager::setSessionManager is not needed.

Reply:
addressed.